blocks/callout/callout.js

```diff
@@ -2,13 +2,23 @@ import { createOptimizedPicture } from '../../scripts/aem.js';
 
 export default function decorate(block) {
   const pic = block.querySelector('picture');
-  const title = block.querySelector('h3');
+  const h3 = block.querySelector('h3');
+  const h1 = block.querySelector('h1');
   const link = block.querySelector('a');
 
-  const textWrapper = title.closest('div');
+  let textWrapper;
+
+  if (h1) {
+    textWrapper = h1.closest('div');
+  } else if (h3) {
+    textWrapper = h3.closest('div');
+  }
+
   textWrapper.classList.add('callout-content');
 
-  link.classList.add('transparent');
+  if (!block.classList.contains('transparent')) {
```

Comment:
I don't understand this condition. why are adding `transparent` class to link when the `transparent` is not specified as block option? won't this add transparent class for all the other variants?

Reply:
nvm

Reply:
I know it can be confusing on a first glance. The combinations of block + button are the following:
- colored callout -> transparent button
- transparent callout -> colored button

So the if/else it's just to apply the `transparent` variant of the button when the block variant is not transparent.